vowpalwabbit/main.cc

```diff
@@ -39,10 +39,12 @@ int main(int argc, char* argv[])
   driver_config.add(make_option("onethread", should_use_onethread).help("Disable parse thread"));
   driver_config.add(make_option("log_level", log_level)
                         .default_value("info")
+                        .hidden()
                         .one_of({"info", "warn", "error", "critical", "off"})
                         .help("Log level for logging messages. Specifying this wil override --quiet for log output"));
   driver_config.add(make_option("log_output", log_output_stream)
                         .default_value("stdout")
+                        .hidden()
```

Comment:
Why are we hiding these?

Reply:
These are only here to get the value. The standard registration place of these is parse args

Reply:
Ahh nice